### PR TITLE
Enable getting R2C/C2R FFT plans from `get_fft_plan()`

### PR DESCRIPTION
**Pull request by leofang**
Address #3263. Follow up of #3102.

### CODE REVIEW

**Comment by leofang** [comment]
Will rebase after #3102 is merged.

**Comment by leofang** [comment]
Conflict is resolved. PTAL.

File: cupyx/scipy/fft/fft.py [hunk 1]
```diff
@@ -318,19 +322,23 @@ def irfft(x, n=None, axis=-1, norm=None, overwrite_x=False, *, plan=None):
         axis (int): Axis over which to compute the FFT.
         norm (None or ``'ortho'``): Normalization mode.
         overwrite_x (bool): If True, the contents of ``x`` can be destroyed.
-        plan (None): This argument is currently not supported.
+        plan (:class:`~cupy.cuda.cufft.Plan1d`) a cuFFT plan for transforming
```
Comment (asi1024) [inline cupyx/scipy/fft/fft.py:325]:
```suggestion
        plan (:class:`~cupy.cuda.cufft.Plan1d` or None) a cuFFT plan for transforming
```

File: cupyx/scipy/fft/fft.py [hunk 2]
```diff
@@ -290,7 +290,14 @@ def rfft(x, n=None, axis=-1, norm=None, overwrite_x=False, *, plan=None):
...
```
Comment (asi1024) [inline cupyx/scipy/fft/fft.py:293]:
```suggestion
        plan (:class:`~cupy.cuda.cufft.Plan1d` or None) a cuFFT plan for transforming
```

File: cupyx/scipy/fft/fft.py [hunk 3]
```diff
@@ -346,7 +354,14 @@ def rfft2(x, s=None, axes=(-2, -1), norm=None, overwrite_x=False, *,
         axes (tuple of ints): Axes over which to compute the FFT.
         norm (None or ``"ortho"``): Keyword to specify the normalization mode.
         overwrite_x (bool): If True, the contents of ``x`` can be destroyed.
-        plan (None): This argument is currently not supported.
+        plan (:class:`~cupy.cuda.cufft.PlanNd`) a cuFFT plan for transforming
```
Comment (asi1024) [inline cupyx/scipy/fft/fft.py:357]:
```suggestion
        plan (:class:`~cupy.cuda.cufft.PlanNd` or None) a cuFFT plan for transforming
```

File: cupyx/scipy/fft/fft.py [hunk 4]
```diff
@@ -375,7 +387,14 @@ def irfft2(x, s=None, axes=(-2, -1), norm=None, overwrite_x=False, *,
...
```
Comment (asi1024) [inline cupyx/scipy/fft/fft.py:390]:
```suggestion
        plan (:class:`~cupy.cuda.cufft.PlanNd` or None) a cuFFT plan for transforming
```

File: cupyx/scipy/fft/fft.py [hunk 5]
```diff
@@ -405,7 +421,14 @@ def rfftn(x, s=None, axes=None, norm=None, overwrite_x=False, *, plan=None):
...
```
Comment (asi1024) [inline cupyx/scipy/fft/fft.py:424]:
```suggestion
        plan (:class:`~cupy.cuda.cufft.PlanNd` or None) a cuFFT plan for transforming
```

File: cupyx/scipy/fft/fft.py [hunk 6]
```diff
@@ -437,7 +457,14 @@ def irfftn(x, s=None, axes=None, norm=None, overwrite_x=False, *, plan=None):
...
```
Comment (asi1024) [inline cupyx/scipy/fft/fft.py:460]:
```suggestion
        plan (:class:`~cupy.cuda.cufft.PlanNd` or None) a cuFFT plan for transforming
```

**Comment by leofang** [comment]
Thanks @asi1024! I fixed the docstrings in 23ac277. Apparently there're several issues (extra `~`, missing `:`, missing double ticks, typo `axis`->`axes`, etc). I hope I didn't miss anything 😅 PTAL.

**Comment by asi1024** [comment]
Jenkins, test this please.

**Comment by asi1024** [comment]
LGTM. Thanks for your PR!

**Comment by leofang** [comment]
Thanks, @asi1024!

**Pull request merged**